README.md

```diff
@@ -1,73 +1,91 @@
     ```
+
     ```bash
         ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDQFVf3TXBOoUyw3GXCRCkM85JYnSx+MT4W7d3bm7RZ7p9xZG9QcQRSvVDg3uc0Fjt2ujzxditMW6epcn5irtByuW55TnKSbCUl9Xl0jlqgCW3GDtP1BCE6xH2iuUjQSSHCFSD7CqjgK4zAdgpOY3r3oZtMoOmHrQDyOGBG4XAQz6LE9u+E3NBLf5STBpR+uvuFAzIimcGVhyeznbyPtUELgHDyNFIlMqzAr7chl8m/x/zm808FLnE3oIUgg+80IbgbF3fmktiNdNgCE6gWwZniVe3avDUX1o2g8BqtYuyFgETugQBy31Y/mZoPfzl2DE7nR8+QF/l4IAW4Owt4XJS5 filipe-macbook 
     ```
+
 - Remember to push everything that you have locally before submitting your challenge. We will only use content that is available in your published repository.
 
-## How to install dependencies
+### How to install dependencies
+
 ```bash
 meteor npm install -g yarn
-meteor yarn install 
+meteor yarn install
 ```
- 
-## How to run
+
+### How to run
+
 ```bash
 meteor yarn start
 ```
 
-## How to run cypress tests
+### How to run tests
+
 ```bash
 meteor yarn cypress
 ```
 
-# What you need to deliver
-A mini-application that allows event hosts to check people into an event. 
+## What you need to deliver
+
+A mini-application that allows event hosts to check people into an event.
 
 The home page should show:
-  - an event selector (`select`) displaying the name of each event by default displaying `Select an event` (communities collection);
-  - a list of people registered in the selected event (people collection).
+
+- an event selector (`select`) displaying the name of each event, by default displaying `Select an event` (communities collection);
+- a list of people registered in the selected event (people collection).
 
 The list of people should allow the event host to:
-  - see first and last name together (full name), company name, title, check-in date, and check-out date both using `MM/DD/YYYY HH:mm` format or `N/A`;
-  - check people into the event clicking in a button "Check-in {person firstName and lastName}";
-  - if she was already checked-in more than 5 seconds ago we want to see a check out button "Check-out {person firstName and lastName}".
+
+- see first and last name together (full name), company name, title, check-in date, and check-out date both using `MM/DD/YYYY HH:mm` format or `N/A`;
+- check people into the event by clicking the "Check-in {person firstName and lastName}" button;
+- if the user was checked-in over five seconds ago, we want to see a "Check-out {person firstName and lastName}" check-out button .
 
 Between the event selector and the list of people we want to see a summary like this:
+
 - `People in the event right now: 10`;
 - `People by company in the event right now: Green Group (10), Hoppe Group (5)`;
-- `People missing check-in: 200`;
+- `People not checked-in: 200`;
```

Comment:
The idea was to not be so explicit and rely on candidate interpretation but maybe `missing check-in` is wrong in English 🤦‍♂